Sources/HTTP/HTTPHeaders.swift

```diff
@@ -56,7 +56,7 @@ extension HTTPHeaders: ExpressibleByDictionaryLiteral {
 }
 
 extension HTTPHeaders {
-    // Used instead of HTTPHeaders to save CPU on dictionary construction
+    /// Used instead of HTTPHeaders to save CPU on dictionary construction
```

Comment:
Isn't that intended to be two `//`, `///` are for generated dox.

Reply:
Ah my mistake